### PR TITLE
Map to new opacities

### PR DESCRIPTION
**Pull request by libertymayc**
Change to opacity scale from design - cc @pseys @dplsek @bhoppers2008 

Change needed for Input and Tabs, so a good time to change
Doesn't impact components directly as defined in palette. Visual change only.

I've tagged this PR in the GH column of each row in the Theme teams doc #1546

### CODE REVIEW

File: site/docs/foundations/opacity.mdx [hunk 1]
```diff
@@ -11,12 +11,14 @@ Opacity refers to the degree of transparency or visibility of a design element.
...
 
-| Token              | Opacity |
-| ------------------ | ------- |
-| `--salt-opacity-1` | 0.15    |
-| `--salt-opacity-2` | 0.25    |
-| `--salt-opacity-3` | 0.40    |
-| `--salt-opacity-4` | 0.70    |
+| Token               | Opacity |
+| ------------------- | ------- |
+| `--salt-opacity-0`  | 0.0     |
```
Comment (libertymayc) [inline site/docs/foundations/opacity.mdx:16]:
```suggestion
| `--salt-opacity-0`  | 0.00     |
```

File: .changeset/cuddly-forks-report.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+| Palette token                              | Old value              | New value               |
+| ------------------------------------------ | ---------------------- | ----------------------- |
+| --salt-palette-opacity-background-readonly | --salt-opacity-1: 0.15 | --salt-opacity-0: 0.00  |
+| --salt-palette-opacity-border-readonly     | --salt-opacity-2: 0.25 | --salt-opacity-8: 0.08  |
+| --salt-palette-opacity-backdrop            | --salt-opacity-4: 0.70 | --salt-opacity-70: 0.70 |
+| --salt-palette-opacity-primary-border      | --salt-opacity-3: 0.40 | --salt-opacity-40: 0.40 |
+| --salt-palette-opacity-secondary-border    | --salt-opacity-2: 0.25 | --salt-opacity-25: 0.25 |
+| --salt-palette-opacity-tertiary-border     | --salt-opacity-1: 0.15 | --salt-opacity-15: 0.15 |
+
```
Comment (alycrys) [inline .changeset/cuddly-forks-report.md:50]: do we need to add `--salt-palette-opacity-disabled` here?

File: .changeset/cuddly-forks-report.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- --salt-opacity-70: 0.70;
++ --salt-opacity-0: 0.00;
++ --salt-opacity-8: 0.08;
++ --salt-opacity-15: 0.15;
++ --salt-opacity-25: 0.25;
++ --salt-opacity-40: 0.40;
++ --salt-opacity-70: 0.70;
+```
+
+Added `--salt-palette-opacity-disabled` of value 0.40 to replace `--salt-palette-opacity-border`, `--salt-palette-opacity-background`, `--salt-palette-opacity-foreground`, `--salt-palette-opacity-fill`, `--salt-palette-opacity-stroke`
```
Comment (joshwooding) [inline .changeset/cuddly-forks-report.md:28]: I don't think the value is important to mention here

File: .changeset/cuddly-forks-report.md (hunk 2 above)
Comment (joshwooding) [reply .changeset/cuddly-forks-report.md:50]: It's mention in the table above

**Review by alycrys — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Review by JoanaMMoreira — APPROVED** [review]

**Pull request merged**